### ISSUE TITLE
Axis direction

### ISSUE THREAD

**Issue by Mahmoodan** [ms609/Ternary#22]
In the current package, I couldn't find a way to change the direction of axis. For examle: when the direction points up, the left axis of the TernaryPlot is horizental and the right axis is inclided toward down . What if I want the right axis be horizental and the left axis to be inclined (correspondigly the bottom axis). It may seems unnecessary from "Plotting" point of view, but when somebody wants to put a 2-D graphical image behind the plot as a background, in which points location is important in the interpretation of the plot, then it needs that the axis direction to be modifiable.

In order to have a better grasp you can compare these two photos:
![photo3](https://user-images.githubusercontent.com/37912601/55330355-1f156e00-5491-11e9-983d-cab6f44145e7.jpg)

![photo2](https://user-images.githubusercontent.com/37912601/55330201-bb8b4080-5490-11e9-836a-3a322b889919.jpg)

**Comment by ms609** [comment]
Glad that you are finding the package useful.  
I think I understand what you are asking, and this should be possible (though a little bit of work, so it might be a wee while before I can implement it).  

I'm not sure that I quite grasp the motivation.  Presumably the ticks on the bottom side of your top picture would to correspond to (and have the same values as) those on the top-left side of the bottom picture?

**Comment by Mahmoodan** [comment]
It is just an idea. If you think that it takes so much time, ignore it. I am not sure how many times it may be used :)
let me explain it in more details:
You have designed the package in a way that it is quite simple to set an image as the background of the TernaryPlot. There are many ternary plots in scientific books that may interest programmers to show the location of points on them (I mean as an output). These images that are taken from external sources are not modifiable (I mean the direction of ticks can not be changed and even rotating them is not the solution, since in many cases it is confusing for the user to see a rotated 2-D diagram. Also there might be some texts in the image and rotating it can cause other issues). 
Suppose that in a book, a plot is presented as the bottom picture and each side of this triangle shows an independent parameter and based on that the regions are showed in a picture in the reference book. Now I want to use that image as the background of my ternaryPlot to show some points on that. Since the direction of ticks can not be changed, then it is not possible to do so.

**Comment by ms609** [comment]
Yes, I can see how it would be simpler in this case to change the position of the axes.

As a workaround in the meantime, if the ticks are already present in the background image, could you simply suppress the ticks on the Ternary plot?  Labelling the axes from 100:0 instead of 0:100 might allow you to put appropriate digits at least in roughly the right place for the ticks.

I'll see what I can do when I next have a spare couple of hours (-:
